project/Build.scala

```diff
@@ -87,6 +87,15 @@ object ScalaCacheBuild extends Build {
     )
     .dependsOn(core)
 
+  lazy val aerospike = Project(id = "scalacache-aerospike", base = file("aerospike"))
+    .settings(implProjectSettings: _*)
+    .settings(
+      libraryDependencies ++= Seq(
+        "com.github.otrimegistro" %% "aerospikez" % "0.2"
```

Comment:
I'm not really happy with adding this lib as a dependency. It hasn't been touched in over 2 years and it depends on a very old version of Scalaz Stream, and it has a lot of transitive dependencies. 

Adding dependencies on unmaintained Scala libs makes it difficult to release ScalaCache when new versions of Scala come out (Scala 2.12 is just around the corner! 🎉 ), and I like to keep transitive dependencies to a minimum, for the sake of ScalaCache's users.

Could you use the [official Aerospike Java client](http://www.aerospike.com/docs/client/java)?